.gitignore

```diff
@@ -1 +1,2 @@
 .DS_Store
+coverage
```

Comment:
I cloned your code and ran the tests and I'm still seeing changes in these files. I think you need to remove the `coverage` directory from the git cache. https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore